### PR TITLE
Document the `no-std` target option in config.toml.example

### PR DESCRIPTION
**Pull request by nicholasbishop**

### CODE REVIEW

File: config.toml.example [hunk 1]
```diff
@@ -721,6 +721,9 @@ changelog-seen = 2
 # probably don't want to use this.
 #qemu-rootfs = <none> (path)
 
+# Skip building the `std` library for this target.
```
Comment (Mark-Simulacrum) [inline config.toml.example:724]: Can we note here that this is default enabled for targets containing -none, nvptx, switch, and uefi in their name?

Comment (nicholasbishop) [reply config.toml.example]: Done

**Pull request merged**